Sources/PackageLoading/ModuleMapGeneration.swift

```diff
@@ -55,8 +55,12 @@ public struct ModuleMapGenerator {
     /// The clang module to operate on.
     private let module: ClangModule
 
-    public init(for module: ClangModule) {
+    /// The file system to be used.
+    private var fileSystem: FileSystem
```

Comment:
I think I have realized that mutability does work for `FileSystem` and am going to remove it, but I haven't made up my mind completely. If we do, this var will go away (this is just a note to myself, I guess).